protocol/communities/manager.go

```diff
@@ -1227,7 +1227,15 @@ func (m *Manager) DeleteCommunityTokenPermission(request *requests.DeleteCommuni
 }
 
 func (m *Manager) reevaluateCommunityMembersPermissions(communityID types.HexBytes) error {
+	// Publish when the reevluation started since it can take a while
+	signal.SendCommunityMemberReevaluationStarted(types.EncodeHex(communityID))
```

Comment:
It would be great if we could use `MessengerSignalsHandler` instead. Globals are bad :police_officer: 

Example:
```golang
if m.config.messengerSignalsHandler != nil {
	m.config.messengerSignalsHandler.MessengerResponse(response)
}
```